public_officials/static/public_officials/js/user_show.js

```diff
@@ -0,0 +1,28 @@
+
+function displayTweets(){
+  $('#tweet-box').html('')
+  let twitterId = this.getAttribute('data-twitter-id')
+  let tweetLink = `<a class="twitter-timeline" data-width="450" href="https://twitter.com/${twitterId}">Tweets by ${twitterId}</a>`
+  $('#tweet-box').append(`<script src="//platform.twitter.com/widgets.js" charset="utf-8"></script>`)
+  $('#tweet-box').append(tweetLink)
+}
+
+function unfollowLegislator(){
+  let legId = this.getAttribute('data-leg-id')
+  let uId = this.getAttribute('data-user-id')
+  let unfollowButton = $(this)
+  removeLegislator(unfollowButton)
+  $.post('/remove-follower', {lid: legId, uid: uId})
+}
+
+function removeLegislator(unfollowButton){
+  unfollowButton.prev().prev().remove()
+  unfollowButton.prev().remove()
+  unfollowButton.remove()
```

Comment:
Those lines are taking the link to the legislators show page, display tweets button, and unfollow button off the page when the button is clicked.